torch/nn/parallel/scatter_gather.py

```diff
@@ -1,7 +1,15 @@
 import torch
 from ._functions import Scatter, Gather
+import warnings
+
+__all__ = ['scatter', 'scatter_kwargs', 'gather']
 
 def is_namedtuple(obj):
+    # Check if type was created from collections.namedtuple or a typing.NamedTuple.
+    warnings.warn("is_namedtuple is deprecated, please use the python checks instead")
```

Comment:
curious, what are the python checks?

Reply:
I guess I just meant that don't rely on `is_namedtuple` API and perform the checks on your own since it's a simple one line function that just calls into python functions like `isinstance` and `hasattr`